electron/scripts/child.js

```diff
@@ -69,6 +70,33 @@ function fillMedicalConditions(all_conditions, child_conditions) {
 		select_box.options.add(opt);
 	}
 	jQuery(".chosen-select").trigger('chosen:updated'); // Needed because chosen-select doesn't update ui automatically like regular select
+	g_child_medical_conditions = child_conditions;
+}
+function updateMedicalConditions() {
+	let selected = jQuery('#medical_condition').chosen().val().map(function(x) {return Number(x);});
+	let child_medical_cond = g_child_medical_conditions.map(function(x) {return x.medical_condition_id;});
+	let added = selected.filter(function(x) {return !child_medical_cond.includes(x);});
+
+	let to_remove = [];
+	for (let i=0; i<g_child_medical_conditions.length; i++) {
```

Comment:
oh "of". I tried "in" and that also grabs all other attributes like functions for the array and stuff. I'll change that